data/intuos-pro-2-s.tablet

```diff
@@ -32,13 +32,13 @@
 [Device]
 Name=Wacom Intuos Pro S
 ModelName=PTH-460
-DeviceMatch=usb:056a:0392;bluetooth:056a:0393;usb:056a:03dc;bluetooth:056a:03dd;
+DeviceMatch=usb:056a:0392;bluetooth:056a:0393;
```

Comment:
@Joshua-Dickens You probably accidentally removed the newer Intous Pro Small

Reply:
Seems like I did, I think I know what happened. Fixed in a quick commit.